.travis.yml

```diff
@@ -1,48 +1,8 @@
 dist: xenial
-addons:
-  postgresql: '9.6'
 language: java
 jdk:
-- openjdk8
-services:
-- mysql
-- postgresql
-- docker
+  - openjdk8
```

Comment:
We can remove this file and disable in TravisCI.com once we move this to actions